src/components/datepicker/dateLocaleProvider.js

```diff
@@ -158,6 +158,10 @@
         if (!date) {
           return '';
         }
+				
+        if (typeof date === 'string') {
```

Comment:
How else would you do you propose to the the date value/string to the DatePicker instance (except my $scope or `controllerAs` references?